ccdproc/image_collection.py

```diff
@@ -216,6 +222,13 @@ def files(self):
         """
         return self._files
 
+    @property
+    def ext(self):
+        """
+        str or int, The extension from which the header and data will be read in all files.
```

Comment:
That's what I said in https://github.com/astropy/ccdproc/pull/463#discussion_r106825591, right? However the important question is: does it also work for the `_generator`? If not we don't need to bother advertising it, but if it works it should be included.